ui/admin/tests/integration/components/worker-diagram/dual-filter/index-test.js

```diff
@@ -5,75 +5,72 @@ import { hbs } from 'ember-cli-htmlbars';
 import { setupIntl } from 'ember-intl/test-support';
 
 module(
-  'Integration | Component | worker-diagram/dual-filter',
+  'Integration | Component | worker-diagram | dual-filter',
   function (hooks) {
     setupRenderingTest(hooks);
     setupIntl(hooks);
 
-    test('it renders the correct diagram when egressFilter is false and ingress is false', async function (assert) {
-      assert.expect(4);
+    let featuresService;
+    const targetWorkerFilterIngress = 'target-worker-filters-v2-ingress';
+    const targetWorkerFilterHCP = 'target-worker-filters-v2-hcp';
+
+    hooks.beforeEach(function () {
+      featuresService = this.owner.lookup('service:features');
+      featuresService.enable(targetWorkerFilterIngress);
+      featuresService.disable(targetWorkerFilterHCP);
+    });
+
+    test('it renders the correct language when egressFilter is false and ingress is false', async function (assert) {
+      assert.expect(2);
       await render(
         hbs`<WorkerDiagram::DualFilter @egressFilter={{false}} @ingressFilter={{false}} />`
       );
 
-      assert.dom('[data-test-dual-filter-egress-off-ingress-off]').isVisible();
       assert
-        .dom('[data-test-dual-filter-egress-on-ingress-off]')
-        .doesNotExist();
+        .dom('.ordered-series-diagram-item:nth-child(2)')
+        .hasText('Any worker');
```

Comment:
Apologies, my previous comment about these tests might not have been worded correctly. We want to avoid testing against raw strings. We should be using the `data-test-*` attributes just to confirm the correct diagram is rendered.